### DISCUSSION TITLE
Enforce minimum number of virtual slides

### DISCUSSION THREAD

**Discussion by markedwards** [nolimits4web/swiper#6030] [General]
When using virtual slides, fewer slides are rendered when at the start or end of the set.  For instance, on the first slide, only 2 slides are rendered (the first slide and the next slide).  When scrolling to the second slide, 3 slides are rendered (the first slide, second slide, and a new third slide).

Is it possible to change this behavior to always have a fixed number of slides rendered?  The behavior I would like is:

activeIndex 0 -- [0, 1, 2]
activeIndex 1 -- [0, 1, 2]
activeIndex 2 -- [1, 2, 3]
etc.

In other words, I want to change the addSlidesAfter or addSlidesBefore when reaching the start or end of the set.
